src/components/WwwFrame/TheHeader.vue

```diff
@@ -363,20 +365,32 @@
 							</router-link>
 
 							<!-- Log in Link -->
-							<kv-button
```

Comment:
Rather than swapping between a `kv-button` and an `a`, you could try changing the styling of the button using the `variant` prop:
```
:variant="isMobile ? 'secondary' : 'link'"
```
and apply whatever tailwinds classes are necessary to get the right styling after that.